dashboard/config/scripts/netsim.script

```diff
@@ -3,5 +3,19 @@ description_short ""
 description ""
 hidden 'true'
 
+stage 'Sending Bits'
+level 'blockly:NetSim:unit1_lesson04'
```

Comment:
Also discovering that an unfortunate side-effect of using a `.level` file is that I have to specify everything using native types, instead of my enums; and I can't comment my JSON.